shell/packages/sandstorm-accounts-ui/LICENSE

```diff
@@ -0,0 +1,20 @@
+forked from Meteor's accounts-ui from https://github.com/meteor/meteor/tree/ab9ef40d258bbb9b9de453600495c8337577c565/packages/accounts-ui-unstyled
```

Comment:
(I'm less worried about the fork of accounts-oauth as that's only temporary and I saw the changes that went upstream.)